src/player_compprocs.c

```diff
@@ -472,10 +472,11 @@ long computer_get_room_role_total_capacity(struct Computer2 *comp, RoomRole rrol
 long computer_get_room_kind_free_capacity(struct Computer2 *comp, RoomKind room_kind)
 {
     struct Dungeon* dungeon = comp->dungeon;
-    if (room_kind == RoK_GARDEN) {
+    if (room_role_matches(room_kind, RoRoF_CrPoolSpawn))
```

Comment:
poolspawn is a portal, not a hatchery, you might be looking for RoRoF_FoodStorage